rtcpeerconnection.go

```diff
@@ -145,8 +166,11 @@ func New(configuration RTCConfiguration) (*RTCPeerConnection, error) {
 		mediaEngine:        DefaultMediaEngine,
 		sctpTransport:      newRTCSctpTransport(),
 		dataChannels:       make(map[uint16]*RTCDataChannel),
+		pclog: &RTCPeerLogger{},
 	}
 
+	pc.pclog.setupLogger(1)
```

Comment:
This is a FANTASTIC idea, nice work!

This way users of the library can switch logging levels at anytime on the fly (and not even supply their own logger) I would change the name though so people know they can call it at anytime though `setLogLevel` maybe?